src/utils/strings.ts

```diff
@@ -35,3 +35,20 @@ export const cyan = "\x1b[36m" + reset;
 export const displayHint = function (message: string) {
     return `${colors.green(`Hint:`)} ${message}`;
 };
+
+/**
+ * Replace an expression of format <prefix>${{<variable>}}<suffix>
+ * to <prefix><value><suffix>
+ *
+ * ${{<variable>}} can be any alphanumeric string with special characters
+ * like /, -, ., _ and whitespace
+ *
+ * @param {string} resource - The original resource string
+ * @param {string} value - The value to replace the placeholder with.
+ * @returns {string} - The updated string with the placeholder replaced.
+ */
+export function replaceExpression(expression: string, value: string) {
```

Comment:
What happens if there are multiple variables in the same expression string? You would only replace the first one, right?

Reply:
Yes, we don't support multiple expressions in the same string, at least for now seems there is no use case.

Reply:
For these cases

```
beginning${{<expression_1>}}something${{<expression_2>}}final
```

It will expand to
```
beginning<value>something${{<expression_2>}}final
```